string picked_bottle
---
bool success
string message
